src/routes/auth/login/+page.svelte

```diff
@@ -4,34 +4,113 @@
     Authentication-only pages should be redirected here if no user session exists.
     @see https://kit.svelte.dev/docs/form-actions
     */
-  import { redirect } from '@sveltejs/kit';
+  import { redirect } from "@sveltejs/kit";
 
-  /** @type {import('./$types').PageData} */
+  /** @type {import("./$types").PageData} */
   export let data;
 
-  /** @type {import('./$types').ActionData} */
+  /** @type {import("./$types").ActionData} */
   export let form;
 
+
   if (data.user) {
-    // If the user is already logged in, redirect them to the home page
-    throw redirect(307, '/');
+    console.log("Logged in already");
+    throw redirect(302, "/posts");
+    window.location.href = "/posts";
```

Comment:
My mistake, `redirect()` is for server code only. The correct function to use is `goto()` - https://kit.svelte.dev/docs/modules#$app-navigation-goto

Reply:
go-to causes errors for some reason, in the rendering of the page, I'll look more into this